src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/processors/hmcts/CourtCaseEventsProcessor.kt

```diff
@@ -63,12 +63,13 @@ class CourtCaseEventsProcessor(
     uniqueDefendants.forEach { defendant ->
       val person = Person.from(defendant)
       telemetryService.trackEvent(
-        HMCTS_MESSAGE_RECEIVED,
+        COURT_MESSAGE_RECEIVED,
         mapOf(
           EventKeys.PNC to person.otherIdentifiers?.pncIdentifier.toString(),
           EventKeys.CRO to person.otherIdentifiers?.croIdentifier.toString(),
           EventKeys.EVENT_TYPE to COMMON_PLATFORM_HEARING.name,
           EventKeys.MESSAGE_ID to sqsMessage.messageId,
+          EventKeys.SOURCE_SYSTEM to SourceSystemType.HMCTS.name,
```

Comment:
Not for now but I wonder if we should rename this to Common Platform in the future?
HMCTS is the Courts and Tribunals Service so covers both Common Platform and Libra (and much more)

Reply:
Yeah i think that makes sense to do, it is getting slightly confusing.